frontend/src/routes/User.js

```diff
@@ -66,8 +96,44 @@ class User extends Component {
     });
   };
 
+  updateForetagsbiljetter = (foretagsbiljetter) => {
+    this.setState({
+      editedUser: {
+        ...this.state.editedUser,
+        foretagsbiljetter: foretagsbiljetter
+      }
+    })
+  };
+
+  editedForetagsbiljett = (index,  { target: { value } }) => {
+    let foretagsbiljetter = this.state.editedUser.foretagsbiljetter;
+    if(foretagsbiljetter.length > index) {
+      foretagsbiljetter[index] = value
+    }
+    this.updateForetagsbiljetter(foretagsbiljetter)
+  };
+
+  addForetagsbiljett = () => {
+    let foretagsbiljetter = this.state.editedUser.foretagsbiljetter;
+    foretagsbiljetter.push("");
+    this.updateForetagsbiljetter(foretagsbiljetter)
+  };
+
+  removeForetagsbiljett = (i) => {
+    let foretagsbiljetter = this.state.editedUser.foretagsbiljetter.filter((ftg, index) => index !== i);
+    this.updateForetagsbiljetter(foretagsbiljetter);
+  };
+
+  trim = (v) => {
```

Comment:
Extract to lib?

Check for array, if so, loop instead of check for `trim` function :)